.npmignore

```diff
@@ -0,0 +1,9 @@
+# Ignore everything
+*
+
+# Publish from an allowlist
+src/**/*.js
+!src/**/*.test.js
```

Comment:
This doesn't seem to exclude the .test.js file properly. I ran `npm pack` and it was still there. I'm also thinking like there is not much unnecessary files in the npm package, so this is not super important. But I'm willing to merge it if you get this test exclusion working.